adafruit_circuitplayground/express.py

```diff
@@ -99,6 +100,16 @@ def __init__(self):
         self._sample = None
         self._sine_wave = None
 
+        # Define sound sensor/mic:
+        try:
+            self._mic = audiobusio.PDMIn(board.MICROPHONE_CLOCK, board.MICROPHONE_DATA,
+                                         sample_rate=16000, bit_depth=16)
+        except TypeError:
+            self._mic = audiobusio.PDMIn(board.MICROPHONE_CLOCK, board.MICROPHONE_DATA,
+                                         frequency=16000, bit_depth=16)
+        self._samples = array.array('H', [0] * 160)
```

Comment:
320 bytes is a lot to hang on to. Are you sure its worth always using it?

Reply:
I moved the array creation to inside the `sound_level` property.